crux-mir-comp/src/Mir/Cryptol.hs

```diff
@@ -382,32 +412,147 @@ munge sym shp0 rv0 = do
     go shp0 rv0
 
 
+-- | Information about a typechecked Cyrptol import
```

Comment:
When you say "a typechecked Cryptol import", do you mean an imported _function_? (The name "`CryFunArgs`" would lead me to believe that this is the case, but I wanted to check.)

A similar question applies to the Haddocks for `CryFunArgs'`.